app/javascript/components/widgets/guests/panel.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import styled from 'styled-components';
+import { parseDate } from '../../../lib/datetime';
+import { colors, weights, fontSizes, spacing } from '../../../lib/theme';
+
+const LoadingMessage = () => <p>Loading...</p>;
```

Comment:
@theroyalblah @mariana0pachon Just pointing out that Simon's work in https://github.com/mojotech/helios2/pull/90 introduces new global `LoadingMessage` and `ErrorMessage` components. If this PR is merged first, Simon can update his PR to update these components. If Simon's PR is merged first, Mariana can update this commit to make use of the new `LoadingMessage` and `ErrorMessage` components. Sound good?